pandas/tests/io/test_fsspec.py

```diff
@@ -181,6 +182,7 @@ def test_arrowparquet_options(fsspectest):
 
 @td.skip_array_manager_not_yet_implemented  # TODO(ArrayManager) fastparquet
 @td.skip_if_no("fastparquet")
+@pytest.mark.xfail(PY310, reason="fastparquet failing on 3.10")
```

Comment:
do we have an issue for this?

Reply:
Created https://github.com/pandas-dev/pandas/issues/45605

Reply:
can you add this issue number to the msgs :->